### ISSUE TITLE
Objects too bright in Three.js r152dev

### ISSUE THREAD

**Issue by andreasrosdal** [mrdoob/three.js#25917]
### Description

Objects loaded from Gltf too bright in Three.js r152dev.

### Reproduction steps

This is a change in Three.js r152 dev version.
Could it be related to the colour space changes going on?

### Code

https://github.com/fciv-net/fciv-net/tree/main/freeciv-web/src/main/webapp/javascript/webgl

### Screenshots

![bilde](https://user-images.githubusercontent.com/41650711/234065803-ee3a6e52-2043-4b3e-b098-40e905b4bb4d.png)

### Version

r152dev

### Device

Desktop

### Browser

Chrome

**Comment by Mugen87** [comment]
> Could it be related to the colour space changes going on?

Potentially, yes. Can you please share the code that shows how you setup and configure your renderer?

Sharing the entire code base of your app is not appropriate since we won't have the resources to study your implementation. So please us the fiddles in the issue template to demonstrate issues.

**Comment by donmccurdy** [comment]
Hi @andreasrosdal – as you noticed, there are a lot of color management changes in r152. These are intentional, and because you aren't already using `renderer.outputEncoding = sRGBEncoding`, they will lead to some lighting differences. I've written a longer guide to the migration here:

https://discourse.threejs.org/t/updates-to-color-management-in-three-js-r152/50791

Hopefully the guide will answer most of your questions, but feel free to follow up in the forums with further questions, or we can reopen this if there does appear to be a bug.

Aside — I noticed you're using Reflector.js and Refractor.js. I have not had time to update those examples yet, and it's possible they will require changes. There is an option to opt-out of these changes (other than the renamed properties) if you prefer.

**Comment by mrdoob** [comment]
@andreasrosdal Please let us know what did you have to change to restore your colors when you figure it out.

**Comment by andreasrosdal** [comment]
I have now updated FCIV.NET to Three.js r152, and I restored the colours. I would have liked to enable ColorManagement to get the most realistic and beautiful colours. However, for now I chose to opt-out of ColorManagement.

I would very much appreciate any advice about how to improve FCIV.NET, specifically how to improve performance and make the game more beautiful. I think full support for WebGPU in Three.js will help. Further, I wish there was better API support for instancing of objects loaded with GLTFLoader, so that game units loaded as GLTF objects would be instantiated  only once, possibly there is an example of how to implement this.

This is what FCIV.NET looks like now:
![bilde](https://user-images.githubusercontent.com/41650711/234957968-2b2bc5f3-fd0b-4354-a59c-e99a3b082184.png)
